source/docs/useful-resources.rst

```diff
@@ -152,6 +152,8 @@ Programming
 
 `Intro to Control Theory <https://blog.wesleyac.com/posts/intro-to-control-part-zero-whats-this>`_ --- A series of blog posts about control theory.
 
+`CTRL ALT FTC <https://www.ctrlaltftc.com/>`_ --- Created by FTC #19376 Thermal Equilibrium, this guide is the most comprehensive and accessible control theory resource for First Tech Challenge teams.
```

Comment:
```suggestion
`CTRL ALT FTC <https://www.ctrlaltftc.com/>`_ --- A guide to control theory created by FTC #19376 Thermal Equilibrium.
```
I would hesitate calling anything "the most comprehensive" control theory resource for FTC teams given Tyler's book exists.

Reply:
This resource should be added just below `Controls Engineering in the FIRST Robotics Competition`, as they are sorted alphabetically.